orchestrator/graphql/utils/to_graphql_result_page.py

```diff
@@ -18,6 +18,6 @@ def to_graphql_result_page(items: list[Any], first: int, after: int, total: str)
             has_next_page=has_next_page,
             start_cursor=start_cursor,
             end_cursor=end_cursor,
-            total_items=int(total) if total else None,
+            total_items=total if total else None,
```

Comment:
```suggestion
            total_items=total,
```

Otherwise `0` is returned as `None`

Reply:
Yep, so that was wrong in the first place

Reply:
Well the old code had a string coming in, so `"0"` would not be converted to None there. Not sure what changed on the calling side but either way it's a bit weird to have `total_items=None`